<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addtask</class>
 <widget class="QDialog" name="addtask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add &amp; Edit</string>
  </property>
  <property name="styleSheet">
   <string notr="true">addtask
{
	background-color:#2D142C;
}
QPushButton
{
	background-color:#801336;
	border-radius:12px;
	border:2px solid #510A32;
}
QPushButton:hover
{
	color: #FFFFFF;
	background-color: #510A32;
}
QLabel
{
	color:#C72C41;
}
QPlainTextEdit
{
	background-color:#510A32;
	color:#C72C41;
	border-radius:12px;
}
QPlainTextEdit:hover
{
	background-color:#801336;
}

QCheckBox
{
	color:#C72C41;
}
QCheckBox:hover
{
	background-color:#C72C41;
	color:#000000;
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_head">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_title">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_title">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_group">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Group</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_group">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>751</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_description">
    <item>
     <widget class="QLabel" name="label_description">
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>105</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_description">
      <property name="minimumSize">
       <size>
        <width>620</width>
        <height>220</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>620</width>
        <height>220</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>751</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layout_steps">
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_steps">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Steps</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="pte_step3">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPlainTextEdit" name="pte_step4">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPlainTextEdit" name="pte_step2">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="pte_step1">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="pte_step5">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPlainTextEdit" name="pte_step6">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>651</y>
     <width>591</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_note">
    <item>
     <widget class="QLabel" name="label_note">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Note</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_note">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_deadline">
    <item>
     <widget class="QLabel" name="label_Line">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DeadLine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Y">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_year">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_M">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_month">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_D">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_day">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_h">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>h</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_hour">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_m">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>m</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_minute">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>590</y>
     <width>101</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_priority">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_priority">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Priority</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>750</y>
     <width>248</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_btn">
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
